### PR TITLE
[WIP] INTLY-600 updates to integreatly webapp to include prometheus metrics

### PR DESCRIPTION
**Pull request by Nanyte25**
## Motivation

https://issues.jboss.org/browse/INTLY-600
## What
 Update to webapp to expose metrics endpoint to support Prometheus monitoring and metrics.

## Why
This is a needed and supported feature. 

## How

By including the node lib prom-client and defining the metrics end-point.

## Checklist:

- [x] Code has been tested locally by PR requester
- [ ] Changes have been successfully verified by another team member

## Progress

- [ ] Finished task
- [ ] TODO

### CODE REVIEW

File: metrics.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+// Create a new metric counter
+const counter = new client.Counter({
+	  name: 'metric_name',
+        help: 'metric_help'
+});
+require('prom-client').collectDefaultMetrics({ register });
+
+// New recordcount 
+function recordcount() {
```
Comment (david-martin) [inline metrics.js]: I'd say you can remove this counter and just have the default metrics for now.

File: server.js [hunk 2]
```diff
@@ -76,6 +77,12 @@ app.get('/config.js', (req, res) => {
   }
 });
 
+// new metrics endpoint for Walkthrough applciation.
```
Comment (david-martin) [inline server.js]: Minor thing, but maybe a more timeless comment like `// prometheus metrics endpoint` would be better here

File: metrics.js (hunk 1 above)
Comment (Nanyte25) [reply metrics.js]: @david-martin happy to remove the counter it was just a test. As the other npm module above and will test it on a cluster on see. Happy to go with the alternative after the test.

File: server.js (hunk 2 above)
Comment (damienomurchu) [inline server.js]:
```suggestion
// new metrics endpoint for Walkthrough application.
```
(if leaving the comment)

**Pull request merged**